app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -156,14 +234,15 @@ private String hashBitmap(Bitmap bmp) {
      * Correct some OCR errors in argument where only letters are expected.
      */
     private static String fixOcrNumsToLetters(String src) {
-        return src.replace("1", "l").replace("0", "o");
+        return src.replace("1", "l").replace("0", "o").replace("5", "s").replace("2", "z");
     }
 
     /**
      * Correct some OCR errors in argument where only numbers are expected.
      */
     private static String fixOcrLettersToNums(String src) {
-        return src.replace("O", "0").replace("l", "1").replace("Z", "2");
+        return src.replace("S", "5").replace("s", "5").replace("O", "0").replace("o",
+                "0").replace("l", "1").replace("I", "1").replace("i", "1").replace("Z", "2");
```

Comment:
We might want to lowercase the string in all callers, instead of needing both uppercase and lowercase replacements.